src/mgmt/srv_drpc.c

```diff
@@ -565,6 +565,67 @@ ds_mgmt_drpc_pool_destroy(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
 	mgmt__pool_destroy_req__free_unpacked(req, NULL);
 }
 
+void
+ds_mgmt_drpc_pool_reintegrate(Drpc__Call *drpc_req, Drpc__Response *drpc_resp)
+{
+	Mgmt__PoolReintegrateReq	*req = NULL;
+	Mgmt__PoolReintegrateResp	resp = MGMT__POOL_REINTEGRATE_RESP__INIT;
```

Comment:
(style)  line over 80 characters

Reply:
Could do the assignment away from the variable declaration, right?
Something like:
```
Mgmt__PoolReintegrateResp	resp;

resp = MGMT__POOL_REINTEGRATE_RESP__INIT;
```

Reply:
No it seems that doesn't work because of the macro expansion not being valid when initialized like that.

after some digging I did find a generated function 
`mgmt__pool_reintegrate_resp__init(&resp);`
which seems to do the same thing.

I'll change it to that in a subsequent patch.